guides/source/active_record_basics.md

```diff
@@ -71,105 +83,236 @@ Convention over Configuration in Active Record
+and it's what turns a regular Ruby class into an Active Record model.
 
-To create Active Record models, subclass the `ApplicationRecord` class and you're good to go:
+`ApplicationRecord` is the base class for all Active Record models in your app.
+To create a new model, subclass the `ApplicationRecord` class and you're good to
+go:
 
 ```ruby
-class Product < ApplicationRecord
+class Book < ApplicationRecord
 end
 ```
 
-This will create a `Product` model, mapped to a `products` table at the
-database. By doing this you'll also have the ability to map the columns of each
-row in that table with the attributes of the instances of your model. Suppose
-that the `products` table was created using an SQL (or one of its extensions) statement like:
+This will create a `Book` model, mapped to a `books` table in the database,
+where each column in the table is mapped to attributes of the `Book` class. An
+instance of `Book` can represent a row in the `books` table. The `books` table
+can be created using an SQL statement like this:
 
 ```sql
-CREATE TABLE products (
+CREATE TABLE books (
   id int(11) NOT NULL auto_increment,
-  name varchar(255),
+  title varchar(255),
+  author varchar(255),
   PRIMARY KEY  (id)
 );
 ```
 
-The schema above declares a table with two columns: `id` and `name`. Each row of
-this table represents a certain product with these two parameters. Thus, you
-would be able to write code like the following:
+Database tables in Rails are typically created using [Active Record
+Migrations](#migrations) and not raw SQL. A migration for the `books` table
+above can be generated like this:
+
+```bash
+$ bin/rails generate migration CreateBooks title:string author:string
+```
+
+and results in this:
+
+```ruby
+# Note:
+# The `id` column, as the primary key, is automatically created by convention.
+# Columns `created_at` and `updated_at` are added by `t.timestamps` line.
+
+# /db/migrate/20240220143807_create_books.rb
+class CreateBooks < ActiveRecord::Migration
+  def change
+    create_table :books do |t|
+      t.string :title
+      t.string :author
+
+      t.timestamps
+    end
+  end
+end
+```
+
+The SQL as well as the migration above declare a table with three columns: `id`,
+`title`, and `author`. Each row of this table can be represented by an instance
+of the `Book` class with the same three attributes: `id`, `title`, and `author`.
+You can access a book's attributes like this:
+
+```irb
+irb> book = Book.new
+=> #<Book:0x00007fbdf5e9a038 id: nil, title: nil, author: nil, created_at: nil, updated_at: nil>
+
+irb> book.title = "The Hobbit"
+=> "The Hobbit"
+irb> book.title
+=> "The Hobbit"
+```
+
+NOTE: You can generate the Active Record model class as well as a matching
+migration with this command `bin/rails generate model Book title:string
+author:string`. This creates both `/app/models/book.rb` and
+`/db/migrate/20240220143807_create_books.rb` files.
+
+### Creating Namespaced Models
```

Comment:
This section switches to `Product` instead of `Book` for the example. Should it be changed to `Book` too?

Reply:
@asavageiv sure, would you like to send a PR with the change?

Reply:
Sure

Reply:
https://github.com/rails/rails/pull/51355